### ISSUE TITLE
chmod /var/www fails on samba share

### ISSUE THREAD

**Issue by jegra** [puphpet/puphpet#1631]
During initialization of the machine (first 'vagrant up' that gets run), I hit an error which halts installation:

```
==> default: Info: Class[Apache::Service]: Scheduling refresh of Service[httpd]
==> default: Notice: /Stage[main]/Main/Exec[Create apache webroot]/returns: chown: changing ownership of ���/var/www���: Permission denied
==> default: Error: mkdir -p /var/www &&                 chown root:www-data /var/www &&                 chmod 775 /var/www &&                 touch /.puphpet-stuff/apache-webroot-created returned 1
 instead of one of [0]
==> default: Error: /Stage[main]/Main/Exec[Create apache webroot]/returns: change from notrun to 0 failed: mkdir -p /var/www &&                 chown root:www-data /var/www &&                 chmod 775 /var/www &&                 touch /.puphpet-stuff/apache-webroot-created returned 1 instead of one of [0]
```

[Link to config.yaml] (http://pastebin.com/tF1FUkme)

**Comment by jtreminio** [comment]
Not sure what's going on with your install:

    changing ownership of ���/var/www���

Regardless, this is a permissions issue. Vagrant is attempting to set permissions to the folder but is unable to do so.

Maybe it's a samba issue? not sure.

**Comment by erdemece** [comment]
is there solution for this. I'm having same problem.

**Comment by jtreminio** [comment]
Fixed by d20258d
